controller/helper/process-survey-instances.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+/**
+ * @module controller/helper/process-survey-instances
+ */
+
+/**
+ * Takes in a Survey Instances and processes it to get Complience chart details
+ * @param {Object} surveys - list of survey instances
+ * @returns {Object} Complience chart data
+ */
+function processSurveyInstances (surveys) {
+    return {
+        labels: pickupSubmissionTime(surveys),
+        datasets: calculateTimeLeft(surveys)
+    };
+}
+
+/**
+ * Takes in a Survey Instances and get the dates to be shown on Complience chart
+ * @param {Object} surveys - list of survey instances
+ * @returns {Object} list of dates labels
+ */
+function pickupSubmissionTime (surveys) {
+    const dates = [];
+
+    for (let index = 0; index < surveys.length; index += 1) {
+      dates[index] = formatDate(surveys[index].startTime);
```

Comment:
for reformatting arrays use the `Array.map` method.

reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map